src/test/scala/chiselTests/stage/ChiselStageSpec.scala

```diff
@@ -207,4 +241,53 @@ class ChiselStageSpec extends AnyFlatSpec with Matchers with Utils {
     exception.getStackTrace.mkString("\n") should include("java")
   }
 
+  it should "NOT add a stack trace to an exception with no stack trace" in {
+    val exception = intercept[java.lang.Exception] {
+      (new ChiselStage).emitChirrtl(new UserExceptionNoStackTrace)
```

Comment:
Nit: only `elaborate` should be needed here (though there is no cost to using `emitChirrtl`.

Reply:
It seems that `class ChiselStage` doesn't have `elaborate` 🤦‍♂️